x/gamm/simulation/operations.go

```diff
@@ -144,14 +145,20 @@ func SimulateMsgCreatePool(ak stakingTypes.AccountKeeper, bk stakingTypes.BankKe
 		poolParams := genPoolParams(r, ctx.BlockTime(), poolAssets)
 
 		// TODO: Replace []string{} with all token types on chain.
-		futurePoolGovernor := genFuturePoolGovernor(r, simAccount.Address, []string{})
```

Comment:
Why was this removed?

Reply:
There was a problem with having an empty denom array as an argument on `r.Intn(len(denoms))`. Previously the gamm operations were not executed so this was not found.